<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node">
        <param name="save_images" type="bool" value="false" />
        <param name="save_images_min_dist" type="double" value="150.0" />
        <param name="use_classifier" type="bool" value="true" />
    </node>
</launch>
